JSDemos/Demos/TextArea/Overview/jQuery/index.js

```diff
@@ -32,7 +32,7 @@ $(() => {
     valueExpr: 'name',
     displayExpr: 'title',
     onValueChanged(data) {
-      editingTextArea.option('valueChangeEvent', data.value);
```

Comment:
I think it is mistake, because we deprecated valueChangeEvent for SelectBox & TagBox widgets only